ibis/expr/types/relations.py

```diff
@@ -764,13 +764,13 @@ def __getitem__(self, what):
             # Projection case
             return self.select(what)
 
-        (what,) = bind(self, what)
```

Comment:
Looks good, but was this causing an issue anywhere?

Reply:
Yes, in a doctest that was using `t[pred1 & pred2]`

Reply:
I think this was triggered by passing a selector: `t[s.numeric()]`

Reply:
Covered here https://github.com/ibis-project/ibis/pull/8215